apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts

```diff
@@ -133,6 +125,38 @@ onDocumentReady(() => {
           title: 'Show/hide student identification information',
         },
       },
+      rubricFilter: {
+        html: rubric_data
+          ? html`
+              <div class="btn-group">
+                <button
+                  type="button"
+                  class="btn btn-secondary dropdown-toggle"
+                  data-bs-toggle="dropdown"
+                  name="rubric-item-filter"
+                  data-bs-auto-close="outside"
+                >
+                  <i class="fas fa-filter"></i> Filter by rubric items
+                </button>
+                <div class="dropdown-menu dropdown-menu-end" id="rubric-item-filter-container">
+                  ${rubric_data.rubric_items.map(
+                    (item) => html`
+                      <label class="dropdown-item dropdown-item-marker"
+                        ><input
+                          type="checkbox"
+                          class="js-rubric-item-filter"
+                          value="${item.description}"
```

Comment:
I'm not a fan of using `item.description` here, why not `item.id`? Note that you could potentially have two rubric items with the same description.